ui/components/atoms/A005-Tab/stories.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+
+// Import index.js from the same folder
+import Tab from '.'
+import Icon from '.'
```

Comment:
you shouldn't need to import the icon here - just tell the tab the iconname as per the schema